### PR TITLE
App dev graph

### PR DESCRIPTION
**Pull request by mic16**
a graph that updates every second displaying any info we'd like

### CODE REVIEW

**Review by Vaati3 — DISMISSED** [review]
LGTM

File: App/TabTemplate.cs [hunk 1]
```diff
@@ -181,5 +183,49 @@ private void buttonFile_Click(object sender, EventArgs e)
...
+                    Invoke((MethodInvoker)delegate { UpdateGraph(); });
+                }
+
+                Thread.Sleep(1000);
+            }
+        }
+
+        private void UpdateGraph()
+        {
+            performanceChart.Series["Series1"].Points.Clear();
```
Comment (AntoineMaillard06) [inline App/TabTemplate.cs:211]: Bad name *Series1* is implicit

**Review by AntoineMaillard06 — CHANGES_REQUESTED** [review]
Good job, but please be more explicit

File: App/TabTemplate.cs (hunk 1 above)
Comment (mic16) [reply App/TabTemplate.cs]: Changed to Network

**Review by Vaati3 — APPROVED** [review]
LGTM again

**Review by AntoineMaillard06 — APPROVED** [review]
It's ok thx Mic16

**Review by theocerutti — APPROVED** [review]
LGTM

**Review by MathildeCouston — APPROVED** [review]

**Pull request merged**